### ISSUE TITLE
vertical_parameter_alignment in constructor implementation Swift

### ISSUE THREAD

**Issue by kushak** [realm/SwiftLint#2005]
Hi! Why does the warning after a standard wrap text in a new line??
And how can this be corrected?

```swift
	init(router: ImagePreviewRouterInput,
 	     interactor: ImagePreviewInteractorInput,
	     popUpManager: PopUpManager,
	     sourceType: UIImagePickerControllerSourceType) { }

```

Warning  in each line: 

Vertical Parameter Alignment Violation: Function parameters should be aligned vertically if they're in multiple lines in a declaration. (vertical_parameter_alignment)

**Comment by marcelofabri** [comment]
@kushak can you please fill the issue template?

**Comment by kushak** [comment]
Duplicate of #1573.

**Comment by Yuop** [comment]
kushak ebanii
